staging/src/github.com/kubeedge/mapper-generator/_template/mapper/cmd/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"errors"
+	"os"
+
+	"k8s.io/klog/v2"
+
+	"github.com/kubeedge/mapper-generator/Template/device"
+	"github.com/kubeedge/mapper-generator/Template/pkg/common"
+	"github.com/kubeedge/mapper-generator/Template/pkg/config"
+	"github.com/kubeedge/mapper-generator/Template/pkg/grpcserver"
+	"github.com/kubeedge/mapper-generator/Template/pkg/httpserver"
+	"github.com/kubeedge/mapper-generator/Template/pkg/util/grpcclient"
+	"github.com/kubeedge/mapper-generator/Template/pkg/util/parse"
```

Comment:
Seems we have no `mapper-generator/Template` dir?

Reply:
This is the package name of the mapper template. When the user creates own mapper, it will be replaced by the mapper name entered by the user.